pysmt/cmd/installers/msat.py

```diff
@@ -42,23 +77,60 @@ def __init__(self, install_dir, bindings_dir, solver_version,
 
         self.python_bindings_dir = os.path.join(self.extract_path, "python")
 
+
     def compile(self):
+        # Patching the swig wrapper
```

Comment:
Not a fan of patching the solver here. The first change should go away after upgrading to 5.5.0. What about the new setup.py? We have a dedicated repo to store patches for the solvers, that is the preferred way of doing this.

Reply:
OK, done